Resources/Prototypes/Entities/Structures/Storage/paper_bin.yml

```diff
@@ -75,4 +75,66 @@
     whitelist:
       tags:
       - Document
-      - Write
\ No newline at end of file
+      - Write
+
+- type: entity
+  parent: PaperBin
+  id: PaperBin50
+  suffix: 50
+  components:
+  - type: Bin
+    initialContents:
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
```

Comment:
woah

Reply:
I dunno how bins work this is what the other ones were like

Reply:
Are you fucking kidding me

Reply:
Anyway consider using office paper instead

Reply:
oh is there any difference besides visual?

Reply:
hey wait isn't there a `count` field? look at the plushie crate or something

Reply:
nevermind, why is this a separate component? D:

Reply:
apparently bins are different